README.md

```diff
@@ -139,17 +139,37 @@ Note some things:
 
 - The value for `HostName` is the hostname you got from Serverless.
 - The value for `Condition.KeyPrefixEquals` is whatever you've configured as `SCALED_FOLDER` in the environment variables.
-- The value for `HttpErrorCodeReturnedEquals` might be `403` or `404` based on your bucket and CloudFront settings.
+- The value for `HttpErrorCodeReturnedEquals` might be `403` or `404` based on your bucket. 403 when the objects in the bucket aren't public, 404 when they are.
 
 #### Let's see if it works!
 
 Upload an image to your bucket (make sure it's publicly readable), for example `foobar.jpg`.
 
-Now access this URL and see if it's worked: `https://<BUCKET_URL>/scaled/500x500/foobar.jpg.webp`.  
+Now access this URL and see if it's working: `https://<BUCKET_URL>/scaled/500x500/foobar.jpg.webp`.  
 If not, the first step in debugging is to go to the Lambda function in the AWS Console and check the CloudWatch logs.
 
 Good luck!
 
+### Use a CloudFront failover origin group
```

Comment:
Fijn deze sectie 👍 
Is het misschien nog goed om boven het stuk over Website Configuration even een zinnetje te wijden aan het feit dat je óf Website Configuration kunt gebruiken, óf Cloudfront?
Iemand hoeft niet allebei die stappen uit te voeren.